config/routes.rb

```diff
@@ -2,4 +2,5 @@
   mount Account::Engine,    at: '/'
   mount Group::Engine,      at: '/'
   mount Onboarding::Engine, at: '/'
+  mount Suppliers::Engine,  at: '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.